.github/workflows/manual-web-client.yml

```diff
@@ -53,3 +54,39 @@ jobs:
 
       - name: Deploy
         run: yarn deploy:web-client:${{ github.event.inputs.environment }}
+      - name: Get current lerna version
+        id: lerna-version
+        uses: saulonunesdev/lerna-get-version-action@v1.0.4
+      - name: Create decompressed copies of assets for Sentry
+        shell: bash
+        run: |
+          cd packages/web-client/tmp
+          mkdir -p deploy-dist-uncompressed/assets
+          cd deploy-dist/assets
+          echo *.js *.map | tr ' ' '\n' | xargs -n1 -I{} cp "{}" "../../deploy-dist-uncompressed/assets/{}.br"
+          cd ../../deploy-dist-uncompressed/assets
+          brotli --rm --decompress *.br
+      - name: Determine assets domain
+        id: assets_domain
+        run: |
+          if [$ENVIRONMENT == 'staging']
+          then
+              echo "::set-output name=domain::app-staging.stack.cards"
+          else
+              echo "::set-output name=domain::app.cardstack.com"
+          fi
+        env:
+          ENVIRONMENT: ${{ github.event.inputs.environment }}
```

Comment:
This is a clunky compensation for the lack of a ternary operator, as taken from [here](https://github.community/t/do-expressions-support-ternary-operators-to-change-their-returned-value/18114/2). Is there anywhere the S3 assets domain could be derived from instead? I couldn’t find it defined anywhere.

Reply:
Another less-verbose-but-still-strange approach would be an adaptation of [this](https://github.community/t/do-expressions-support-ternary-operators-to-change-their-returned-value/18114/4):

```
${{ fromJSON('["app-staging.stack.cards", "app.cardstack.com"]')[github.events.input.environment == 'staging'] }}
```